src/entry/labelling/offer-labeller.service.js

```diff
@@ -64,6 +64,8 @@ function OfferLabeller(jobLogger, udbApi, OfferLabelBatchJob, QueryLabelJob, $q)
       .unlabelOffer(offer.apiUrl, labelName)
       .then(function () {
         offer.unlabel(labelName);
+      }).catch(function (error) {
+        throw error;
```

Comment:
```suggestion
      })
```

When no catch provided the error will be thrown further up the chain right?

Reply:
Sometimes not I know I've had this issue multiple times over the years but I've never managed to pinpoint why, so this is the first thing I tried but in the end removing it still works so must have been mostly the interceptor I assume?